src/ports.js

```diff
@@ -239,6 +239,35 @@
             return this;
         },
 
+        /**
+         * @param {string} portId
+         * @param {string} path
+         * @param {*=} value
+         * @param {object=} opt
+         * @returns {joint.dia.Element}
+         */
+        portProp: function(portId, path, value, opt) {
+
+            var index = this.getPortIndex(portId);
+
+            if (index === -1) {
+                throw new Error('Element: unable to find port with id ' + portId);
+            }
+
+            if (!_.startsWith(path, '/')) {
```

Comment:
in this case at least one `/` has to be there, as the `path` is just a part of the 'result' path, 
` var portPath = ['ports/items/', index, path].join('');`